src/views/SubjectAllQuestions/CreateExamModal/CreateExamModal.tsx

```diff
@@ -96,6 +110,15 @@ export const CreateExamModal = ({
               }}
               value={percentage}
             />
+            <div className={classes.learntQuestionsWrapper}>
```

Comment:
The text isn't centred vertically
<img width="259" alt="CleanShot 2023-05-30 at 12 27 26@2x" src="https://github.com/bibixx/zdaj-se-pjatk/assets/14880213/bbaa3e4b-2362-4dad-8469-c1253f00bf46">